chains/evm/contracts/ccvs/CommitRamp.sol

```diff
@@ -1,35 +1,41 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.24;
 
+import {ICCVOffRampV1} from "../interfaces/ICCVOffRampV1.sol";
+
 import {Client} from "../libraries/Client.sol";
 import {MessageV1Codec} from "../libraries/MessageV1Codec.sol";
-import {BaseOnRamp} from "./BaseOnRamp.sol";
+import {BaseOnRamp} from "./components/BaseOnRamp.sol";
+import {SignatureQuorumVerifier} from "./components/SignatureQuorumVerifier.sol";
+
+import {IERC165} from
+  "@chainlink/contracts/src/v0.8/vendor/openzeppelin-solidity/v5.0.2/contracts/utils/introspection/IERC165.sol";
+
 import {Ownable2StepMsgSender} from "@chainlink/contracts/src/v0.8/shared/access/Ownable2StepMsgSender.sol";
 
-/// @notice The CommitOnRamp is a contract that handles lane-specific fee logic.
-/// @dev The OnRamp and OffRamp form a cross chain upgradeable unit. Any change to one of them results in an onchain
-/// upgrade of both contracts.
-contract CommitOnRamp is Ownable2StepMsgSender, BaseOnRamp {
+/// @notice The CommitRamp is a contract that handles lane-specific fee logic and message verification.
+/// @dev OnRamp and OffRamp capabilities are combined to enable a single proxy address for a CCV each chain.
+contract CommitRamp is Ownable2StepMsgSender, ICCVOffRampV1, SignatureQuorumVerifier, BaseOnRamp {
```

Comment:
Can you check the spec for the latest naming for this contract? I don't think this matches

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/1182/commits/8f1762ef84e1348c20ab970fbc61f97ba2327bf7